.github/workflows/sdk-cli-perf-monitor-test.yml

```diff
@@ -36,6 +36,12 @@ jobs:
       - name: checkout
         uses: actions/checkout@v4
 
+      - name: Set Flow Run Id to an Environment Variable
+        run: echo "FLOW_RUN_ID=${{ github.run_id }}" >> $GITHUB_ENV
```

Comment:
Nit: GITHUB_RUN_ID since it's github CI's run id instead of flow's run id?

Reply:
Sure, I have made the necessary modifications.